### PR TITLE
Tellef

### PR DESCRIPTION
**Pull request by tellefs**

### CODE REVIEW

**Comment by tellefs** [comment]
Optimzed the learning rate of XGBoost
